examples/example.js

```diff
@@ -15,7 +15,8 @@ const schema = {
 
 fastify
   .get('/', schema, function (req, reply) {
-    reply(null, { hello: 'world' })
+    reply.header('Content-Type', 'application-json').code(200)
+    reply.reply(null, { hello: 'world' })
```

Comment:
can we call it res instead of reply, so we have `res.reply`?
If this is the case, there is no need to have an error as the first argument.

Another option might be to have:

```
const res = req.reply()
res.header('Content-Type', 'application-json').code(200)
res.body({ hello: 'world' })
res.emit('error', err)
somethingAsync(res.asCallback) // use an accessor here
```